theme/scripts/cart.ts

```diff
@@ -4,20 +4,16 @@ import * as Sentry from "@sentry/browser";
 import handleFetchJSONResponse from './utils/handleFetchJSONResponse';
 import activatable from './activatable';
 
-enum Action {
-  ADD = 'add',
-  REMOVE = 'remove'
-};
-
 export default(function() {
   const Cart = {
     SELECTORS: {
-      addToCartForm: "[data-add-to-cart-form]",
-      addToCartButton: "[data-add-to-cart-button]",
-      quantitySelector: "[data-quantity-selector]",
+      productAddToCartForm: "[data-product-add-to-cart-form]",
+      desktopProductAddToCartButton: "[data-desktop-product-add-to-cart-button]",
+      mobileProductAddToCartButton: "[data-mobile-product-add-to-cart-button]",
```

Comment:
Added because the location of the add to cart button changes on desktop vs mobile (using display:none to show or hide on different screen sizes)

Reply:
lol, was this why we weren't seeing that "added to cart"?